src/Scope.ts

```diff
@@ -850,6 +862,52 @@ export class Scope {
         });
         return results;
     }
+
+    public getAllClassMemberCompletions() {
+        let results = new Map<string, CompletionItem>();
+        let filesSearched = new Set<BscFile>();
+        for (const file of this.getAllFiles()) {
+            if (isXmlFile(file) || filesSearched.has(file)) {
+                continue;
+            }
+            filesSearched.add(file);
+
+            const statementHandler = (s: ClassMethodStatement) => {
+                if (!results.has(s.name.text) && s.name.text.toLowerCase() !== 'new') {
+                    results.set(s.name.text, {
+                        label: s.name.text,
+                        kind: CompletionItemKind.Method
+                    });
+                }
+            };
+            const fieldStatementHandler = (s: ClassFieldStatement) => {
+                if (!results.has(s.name.text)) {
+                    results.set(s.name.text, {
+                        label: s.name.text,
+                        kind: CompletionItemKind.Field
+                    });
+                }
+            };
+            file.parser.ast.walk(createVisitor({
+                ClassMethodStatement: statementHandler,
+                ClassFieldStatement: fieldStatementHandler
+            }), {
+                walkMode: WalkMode.visitStatements
+            });
+
+        }
+        return results;
+    }
+
+    public getClassHieararchy(className: string) {
```

Comment:
typo - Should be `getClassHierarchy`

Reply:
unless you're a pirate!!!.. erm. yes. okay you got me.. thanks :)